xio-core/src/test/java/com/xjeffrose/xio/client/ClientConfigTest.java

```diff
@@ -0,0 +1,54 @@
+package com.xjeffrose.xio.client;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import com.xjeffrose.xio.SSL.TlsConfig;
+import io.netty.channel.ChannelOption;
+import java.net.InetSocketAddress;
+import java.util.Map;
+import org.assertj.core.util.Maps;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ClientConfigTest extends Assert {
+  @Test
+  public void testBuilderCustomValues() {
+    Config config = ConfigFactory.load();
+    ClientConfig fallbackObject = new ClientConfig(config.getConfig("xio.clientTemplate"));
+    Map expectedBootstrapOptions = Maps.newHashMap(ChannelOption.AUTO_READ, new Object());
+    String expectedName = "name";
+    TlsConfig expectedTls = new TlsConfig(ConfigFactory.load("tls-reference"));
```

Comment:
I'm trying to avoid having to make a builder for TlsConfig for now in hopes that we won't need one. If we need to be able to create different ones at runtime, we can add it then

Reply:
You shouldn’t need a builder for TlsConfig. We’ll always want something typesafe config based so we can pull in defaults from reference.conf

Reply:
That's what I was hoping for 👍